spring-cloud-gateway-mvc/src/main/java/org/springframework/cloud/gateway/mvc/ProxyExchange.java

```diff
@@ -130,6 +130,7 @@
  * </p>
  *
  * @author Dave Syer
+ * @author chentong
  *
  */
 public class ProxyExchange<T> {
```

Comment:
This is not part of spring-cloud-gateway-server and should be removed from this enhancement since this module doesn't know anything about server properties.

Reply:
understand,i will remove this.